compiler/damlc/tests/daml-test-files/Sha256.daml

```diff
@@ -1,9 +1,8 @@
 module Sha256 where
 
+import Daml.Script
 import DA.Assert
 import DA.Text as T
 
-test = scenario do
+test = script do
   T.sha256 "foo" === "2c26b46b68ffc68ff99b453c1d30413413422d706483bfa0f98a5e886266e7ae"
```

Comment:
Do we really need to test that at integration level ?

Reply:
I'm not sure I can answer this question. I'd assume not? But theres maybe a possibility for the input string to get modified in some way before getting to wherever it's actually hashed? I feel like that'd be a more prevalent issue though

Reply:
OK. Let it there for now.